docusaurus/docs/metadata.md

```diff
@@ -79,6 +80,29 @@ Grafana Enterprise specific features.
 | ------------------------- | ------- | -------- | ------------------------------------------------------------------- |
 | `healthDiagnosticsErrors` | boolean | No       | Enable/Disable health diagnostics errors. Requires Grafana >=7.5.5. |
 
+## iam
+
+Identity and Access Management section. Grafana will read this section and initialize a service account for the plugin, with a tailored set of
+[Grafana RBAC permissions](https://grafana.com/docs/grafana/latest/administration/roles-and-permissions/access-control/custom-role-actions-scopes/#rbac-permissions-actions-and-scopes).
+Then its bearer token will be share to the plugin backend using the `GF_PLUGIN_APP_CLIENT_SECRET` environment variable.
+
+Requires Grafana >= 10.3.0. Currently behind a feature toggle `externalServiceAccounts`.
```

Comment:
Should I wait for the feature toggle to be removed?

Reply:
Maybe we can link here to the example that uses it?